### PR TITLE
Added qos and partitions details in slurm 

### PR DESCRIPTION
**Pull request by absrocks**

### CODE REVIEW

File: docs/assets/tables/slurm_qos.csv
Comment (alexpacheco) [inline docs/assets/tables/slurm_qos.csv:1]: --qos=high has a 30 day max waltime mandated by the monthly downtime. We need to stop mentioning that condo investors will have unlimited wall time. That is flat out wrong! Maybe change this the next time you push an update but we need to get this version published and it's ok for now

**Review by alexpacheco — APPROVED** [review]
Approved but see comment on --qos=high

**Comment by alexpacheco** [comment]
Reviewed and merged changes

File: docs/assets/tables/slurm_qos.csv
Comment (absrocks) [reply docs/assets/tables/slurm_qos.csv:1]: Ok, I will then add this to the table.

**Pull request merged**